app/components/Bookings/AlertFlags.js

```diff
@@ -18,96 +21,19 @@ const AlertFlags = (alerts, divClassName, onAlertFlagClick) => {
-        </span>
-      )}
-      {isShown('XSA') && (
-        <span className="assault-status" {...linkOnClick(onAlertFlagClick)}>
-          STAFF ASSAULTER
-        </span>
-      )}
-      {isShown('XA') && (
-        <span className="arsonist-status" {...linkOnClick(onAlertFlagClick)}>
-          <img src="/img/Arsonist_icon.png" className="arsonist-adjust" alt="" width="11" height="14" /> ARSONIST
-        </span>
-      )}
-      {isShown('PEEP') && (
-        <span className="disability-status" {...linkOnClick(onAlertFlagClick)}>
-          <img src="/img/Disability_icon.png" className="disability-adjust" alt="" width="14" height="15" /> PEEP
-        </span>
-      )}
-      {isShown('XEL') && (
-        <span className="elist-status" {...linkOnClick(onAlertFlagClick)}>
-          E&#x2011;LIST
-        </span>
-      )}
-      {isShown('XRF') && (
-        <span className="risk-females-status" {...linkOnClick(onAlertFlagClick)}>
-          RISK TO FEMALES
-        </span>
-      )}
-      {isShown('XTACT') && (
-        <span className="tact-status" {...linkOnClick(onAlertFlagClick)}>
-          TACT
-        </span>
-      )}
-      {isShown('XCO') && (
-        <span className="corruptor-status" {...linkOnClick(onAlertFlagClick)}>
-          <img src="/img/CU_icon.png" className="corruptor-adjust" alt="" width="14" height="15" /> CORRUPTOR
-        </span>
-      )}
-      {isShown('XCA') && (
-        <span className="chemical-attacker-status" {...linkOnClick(onAlertFlagClick)}>
-          CHEMICAL ATTACKER
-        </span>
-      )}
-      {isShown('XCI') && (
-        <span className="concerted-indiscipline-status" {...linkOnClick(onAlertFlagClick)}>
-          CONCERTED INDISCIPLINE
-        </span>
-      )}
-      {isShown('XR') && (
-        <span className="racist-status" {...linkOnClick(onAlertFlagClick)}>
-          RACIST
-        </span>
-      )}
-      {(isShown('RTP') || isShown('RLG')) && (
-        <span className="risk-lgbt-status" {...linkOnClick(onAlertFlagClick)}>
-          RISK TO LGBT
-        </span>
-      )}
-      {isShown('XHT') && (
-        <span className="hostage-taker-status" {...linkOnClick(onAlertFlagClick)}>
-          HOSTAGE TAKER
-        </span>
-      )}
-      {isShown('XCU') && (
-        <span className="controlled-unlock-status" {...linkOnClick(onAlertFlagClick)}>
-          CONTROLLED UNLOCK
-        </span>
-      )}
-      {isShown('XGANG') && (
-        <span className="gang-member-status" {...linkOnClick(onAlertFlagClick)}>
-          GANG MEMBER
-        </span>
-      )}
-      {isShown('CSIP') && (
-        <span className="csip-status" {...linkOnClick(onAlertFlagClick)}>
-          CSIP
-        </span>
-      )}
-      {isShown('F1') && (
-        <span className="veteran-status" {...linkOnClick(onAlertFlagClick)}>
-          VETERAN
-        </span>
-      )}
+      {alertFlags &&
+        alertFlags.map(alert => {
+          return (
+            <>
```

Comment:
Do we have a general approach for these empty containers?  I usually go with `<fragment>` but depends on the team i suppose.

Reply:
They should all be `<>` now @Robc24 as we have `react/jsx-fragments` es lint rule enabled from the airbnb config